packages/auth-providers/auth0/api/package.json

```
@@ -0,0 +1,45 @@
+{
+  "name": "@redwoodjs/auth-auth0-api",
+  "version": "3.2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/redwoodjs/redwood.git",
+    "directory": "packages/auth-providers/auth0/api"
+  },
+  "license": "MIT",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "yarn build:js && yarn build:types",
+    "build:clean-dist": "rimraf 'dist/**/*/__tests__'",
+    "build:js": "babel src -d dist --extensions \".js,.ts,.tsx\" --copy-files --no-copy-ignored",
+    "build:types": "tsc --build --verbose",
+    "build:watch": "nodemon --watch src --ext \"js,ts,tsx,template\" --ignore dist --exec \"yarn build\"",
+    "prepublishOnly": "NODE_ENV=production yarn build",
+    "test": "jest src",
+    "test:watch": "yarn test --watch"
+  },
+  "dependencies": {
+    "@babel/runtime-corejs3": "7.20.1",
+    "core-js": "3.26.0",
+    "jsonwebtoken": "8.5.1",
+    "uuid": "9.0.0"
```

Doesn't seem like we use `uuid` anywhere, but looks like we're missing `jwks-rsa` (I'm just looking at dist to figure out cause it's one file)

```suggestion
    "@babel/runtime-corejs3": "7.20.1",
    "core-js": "3.26.0",
    "jsonwebtoken": "8.5.1",
    "jwks-rsa": "2.0.5"
```

---

Done